### ISSUE TITLE
Embedded Tables

### ISSUE THREAD

**Issue by jjc54** [mine-cetinkaya-rundel/quarto-manuscripts-rmed#1]
I really enjoyed your presentation. Is it possible to embed tables in the manuscript from a source .qmd much like you did with figures? 

The generation of tables can be just as complex as figures when you get into packages like gt and gtsummary. So far I'm having massive errors when attempting this :/

**Comment by mine-cetinkaya-rundel** [comment]
Yes, embedding tables from a supporting notebook is definitely possible. The error could be on the Quarto side or on the table generation side, but I'd need a bit more info / reprex to guide you in the right direction/offer a solution.
